shared/output/cloud/vision_v1/lib/google/cloud/vision/v1/image_annotator/paths.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Auto-generated by gapic-generator-ruby. DO NOT EDIT!
+
+
+module Google
+  module Cloud
+    module Vision
+      module V1
+        module ImageAnnotator
+          # Path helper methods for the ImageAnnotator API.
+          module Paths
```

Comment:
I would expect this to include `location_path`, since all the image annotator requests include a parent of that form.

Reply:
None of this service's [request messages](https://github.com/googleapis/googleapis/blob/master/google/cloud/vision/v1/image_annotator.proto) or the messages that they contain have a field that defines a resource with a type of "locations.googleapis.com/Location". We would need the proto structure to be annotated for that helper to show up using this PR's approach.

Reply:
Oh right, I see.